### DISCUSSION TITLE
After updating RStudio and R language the CSnippets stopped working

### DISCUSSION THREAD

**Discussion by koprifka** [kingaa/pomp#201] [Q&A]
Hi, I was using pomp for about 3 months. I had some problems with installation but everything worked out eventually. But today I updated my RStudio and R (from 4.2.2 to 4.3.1) and CSnippets stopped working. After trying to build the pomp object I get this error message:
```
Error: error in building shared-object library from C snippets: in ‘Cbuilder’: compilation error: cannot compile shared-object library ‘C:/Users/Notebook/AppData/Local/Temp/RtmpmaPeYZ/17296/pomp_af35023f8834bb8821532cdc0a6fcb8f.dll’: status = 1
compiler messages:
Error in system(paste(MAKE, p1(paste("-f", shQuote(makefiles))), "compilers"),  : 
  'make' not found
Calls: <Anonymous> -> .shlib_internal -> system
Execution halted
In addition: Warning message:
In system2(command = R.home("bin/R"), args = c("CMD", "SHLIB", "-c",  :
  running command '"C:/PROGRA~1/R/R-43~1.1/bin/R" PKG_CPPFLAGS="-I"C:/Users/Notebook/AppData/Local/R/win-library/4.3/pomp/include" -I"C:/Users/Notebook/Documents/1vejška/R/Bakalarka"" CMD SHLIB -c -o C:/Users/Notebook/AppData/Local/Temp/RtmpmaPeYZ/17296/pomp_af35023f8834bb8821532cdc0a6fcb8f.dll C:/Users/Notebook/AppData/Local/Temp/RtmpmaPeYZ/17296/pomp_af35023f8834bb8821532cdc0a6fcb8f.c' had status 1
```

On the FAQ website I found some checking codes about this topic. None of them is working for me (didn't get any Hello! :D)

I also tried to switch back to the old R version in global options in R Studio. After this everything seems to be working. So the problem is just with the new version of R.

Any idea how to fix it? I am really scared about the re-installation because of the problems I had before. But I will do what is necessary.

**Comment by kingaa** [comment]
The change from from **R** 4.2 to 4.3 necessitates that you make the corresponding update to your [**Rtools** installation](https://kingaa.github.io/pomp/install.html#important-note-for-windows-users).  It should be straightforward.
